packages/mobile/src/fiatExchanges/utils.tsx

```diff
@@ -59,6 +89,152 @@ export const fetchLocationFromIpAddress = async () => {
   return ipAddressObj
 }
 
+export const fetchSimplexQuote = async (
+  userAddress: string,
+  ipAddress: string,
+  currencyToBuy: CurrencyCode,
+  fiatCurrency: LocalCurrencyCode,
+  fiatAmount: number
+) => {
+  const response = await fetch(`${networkConfig.simplexUrl}/wallet/merchant/v2/quote`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `ApiKey ${networkConfig.simplexApiKey}`,
+    },
+    body: JSON.stringify({
+      end_user_id: userAddress,
+      digital_currency: currencyToBuy,
+      fiat_currency: fiatCurrency,
+      requested_currency: fiatCurrency,
+      requested_amount: fiatAmount,
+      wallet_id: 'valorapp',
+      client_ip: ipAddress,
+      payment_methods: ['credit_card'],
+    }),
+  })
+
+  const simplexQuote: SimplexQuote = await response.json()
+  return simplexQuote
+}
+
+export const createUuidv4 = () =>
```

Comment:
Why are we doing this instead of just using the `uuidv4()` method on the `uuid` package?

Reply:
Didn't know this package existed! Thanks, I'll use it.